cypress/integration/settings/create_dunning_type_spec.js

```diff
@@ -0,0 +1,31 @@
+import { DunningType } from '../../support/utils/dunning_type';
+
+describe('create dunning type', function() {
+  // if creating more than 1 default dunning type will give an error
+  // const timestamp = new Date().getTime();
+  // const dunningTypeName = `dunning test ${timestamp}`;
+
+  // before(function() {
+  //   cy.fixture('settings/dunning_type.json').then(dunningType => {
+  //     Object.assign(new DunningType(), dunningType)
+  //       .setName(dunningTypeName)
+  //       // .setCheckDefault(dunningType.default)
+  //       .apply();
+  //   });
+  // });
+
+  it('operations on BP', function() {
+    cy.visit('/window/123');
```

Comment:
i suggest that you use our `cy.visitWindow()` command instead, because it waits for the API to get the window-data before going on